src/pages/Homepage/Homepage.vue

```diff
@@ -4,24 +4,11 @@
 
 <script>
 /* eslint-disable vue/multi-word-component-names */
-import { gql } from '@apollo/client';
-import experimentAssignmentQuery from '@/graphql/query/experimentAssignment.graphql';
-import experimentVersionFragment from '@/graphql/fragments/experimentVersion.graphql';
 import { preFetchAll } from '@/util/apolloPreFetch';
 import numeral from 'numeral';
 
 const ContentfulPage = () => import('@/pages/ContentfulPage');
```

Comment:
Can we remove this file entirely?

Reply:
This from a slack convo with Stover:

> It should be safe to clean up the old commented out route (leave the redirect in place for now) followed by cleaning up the Homepage.vue file and any homepage specific (non-contentful) components.
> We should keep the Contentful page intact for now

https://kivahq.slack.com/archives/CB7GRAFKK/p1700513842600929

Reply:
I don't think we should remove the Contentful page and related code just yet. But I think it's fine to remove the Homepage.vue file...

Reply:
Gotcha, thanks!